### PR TITLE
Update BOM.txt

### PR DESCRIPTION
**Pull request by JDogHerman**
Still missing P/N for power supply.

### CODE REVIEW

**Comment by BarbourSmith** [comment]
It's a custom piece, but we can specify that to make it more clear. Where would be the place you want to see it written?

**Comment by JDogHerman** [comment]
What are the specs of the power supply? I can look for a retailer who can supply one.

File: BOM.txt [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-Indicator LED:        475-1410-1-ND           2        LED 1-2
-Current Limiting R1:  RMCF0805JT270RCT-ND     1        R1
-Current Limiting R2:  P330ADCT-ND             1        R2
-25x25mm heatsink      Custom                  2        N/A
+Part Name                Notes                 PartNumber              Source        QTY      Desiganator
+XH-Series Connectors     Male Connector        455-2271-ND             DigiKey       3        J1-3
+Power Jack               2x5.5MM DC            CP-002A-ND              DigiKey       1        J4
+1x8 .1" headers          2.54mm Pitch          732-5321-ND             DigiKey       4        J5-8
+2x2 .1" headers          2.54mm Pitch          952-2119-ND             DigiKey       2        J9,10
+Flyback Diodes           400V 1A               GS1G-LTPMSCT-ND         DigiKey       12       D1-12
```
Comment (JDogHerman) [inline BOM.txt:8]: Changed the diode from one mfg to another as it was regularly kept in stock and was also cheaper. 😈

**Comment by mrfugu** [comment]
Please see https://github.com/MaslowCNC/Electronics/wiki/DIY-PCB-Parts-List 

This issue can be closed.

**Pull request closed without merging**